server/app/views/admin/programs/ProgramIndexViewV2.java

```diff
@@ -230,108 +335,74 @@ private static Instant extractLastUpdated(
     return program.lastModifiedTime().orElse(Instant.EPOCH);
   }
 
-  private String extractProgramStatusText(
-      Optional<ProgramDefinition> draftProgram, Optional<ProgramDefinition> activeProgram) {
-    if (draftProgram.isPresent() && activeProgram.isPresent()) {
-      return "Active, with draft";
-    } else if (draftProgram.isPresent()) {
-      return "Draft";
-    } else if (activeProgram.isPresent()) {
-      return "Active";
-    }
-    throw new IllegalArgumentException("Program neither active nor draft.");
+  Tag renderViewLink(ProgramDefinition program) {
+    String viewLink =
+        baseUrl
+            + controllers.applicant.routes.RedirectController.programByName(program.slug()).url();
+    ContainerTag button =
+        ButtonUtils.makeSvgTextButton("View", Icons.VISIBILITY_SVG_PATH)
+            .withClass(AdminStyles.TERTIARY_BUTTON_STYLES);
+    return ButtonUtils.asLinkButton(button, viewLink);
```

Comment:
issue (blocking): this design introduces a regression in that the the program deep link is no longer visible to the admin. The purpose of displaying it in the program list isn't to link to the application for the admin, but rather to display it to the admin so they can copy it for use outside of CiviForm (which is why the link is not clickable in the current UI).

Reply:
> What do you think about "View" instead being changed to something that copies the program link to the clipboard?

that sounds fine, just want it to be super obvious how to get the deep link before switching to the new version of the page